lib/rucio/client/baseclient.py

```diff
@@ -402,6 +410,9 @@ def _send_request(self, url, headers=None, type='GET', data=None, params=None, s
                     result = self.session.delete(url, headers=hds, data=data, verify=self.ca_cert, timeout=self.timeout)
                 else:
                     return
+                if result.status_code in STATUS_CODES_TO_RETRY:
+                    back_off(retry)
```

Comment:
log warning about this and state how long you will wait

Reply:
(and the other instances where this happens as well)

Reply:
done